app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -87,19 +116,45 @@ const Entry = ({
     if (!user.id || user.id === '') {
       return;
     }
-
     getCurrentGameWeek();
     getAllEntries();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can you please explain the purpose of this comment here?

Reply:
> Can you please explain the purpose of this comment here?

Our linting rule is looking for all of the dependencies of `getAllEntries` and without that comment it continues to flag a warning.

However, we do not need all the dependencies, so I added the comment to disalbe that linter check for that line only.

Reply:
Taking a look at your [Loading Spinner PR](https://github.com/LetsGetTechnical/gridiron-survivor/pull/399/files/c57250af0d1cc408950512270ea75e58a50e35fa#diff-04023da7c5cebd3bf7f6cf690a6022e199b0b4d8c85cc9589b331686cab65b2cR62), it seems this comment is unnecessary.

Reply:
> Taking a look at your [Loading Spinner PR](https://github.com/LetsGetTechnical/gridiron-survivor/pull/399/files/c57250af0d1cc408950512270ea75e58a50e35fa#diff-04023da7c5cebd3bf7f6cf690a6022e199b0b4d8c85cc9589b331686cab65b2cR62), it seems this comment is unnecessary.

@choir27 I wanted to bounce back on @shashilo stating that it is unnecessary. While it is only warning and not an _error_ I don't see the harm in adding said comment so as to fill our terminals with less warnings. That is what these comments are there for IMO.

Reply:
@shashilo Your thoughts? Is the comment still unnecessary?

Reply:
@shashilo please advise on the comments above.